cron/rotateIndexes.py

```diff
@@ -81,14 +84,19 @@ def esRotateIndexes():
                         logger.debug('Creating %s index' % newindex)
                         es.create_index(newindex)
                     # set aliases: events to events-YYYYMMDD
-                    # and events-previous to events-YYYYMMDD-1 for example
+                    # and events-previous to events-YYYYMMDD-1
                     logger.debug('Setting {0} alias to index: {1}'.format(index, newindex))
-                    es.create_alias(index, newindex)
+                    alias = index
+                    index = newindex
+                    es.update_alias(oldindex, index, alias)
```

Comment:
Based on the assignment statements, we are functionally passing es.update_alias(oldindex, newindex, index). I get clarifying alias = index, since we are basically passing the date string (eg events-YYYYMMDD) but why the other reassignment?

Reply:
In the abstraction layer we have 3 variables.
However, we have two aliases being used: events, events-previous

I changed the abstraction layer to only take in index, oldindex, and alias.
I couldn't reuse that function if it was taking in oldindex, newindex, previndex, alias, index, etc.

So I separated by an if statement the two calls (one calls for "events" alias update, and the other for "events-previous" update.

The casting is to populate what the abstraction layer expects with the correct values.
I do think this can be done more elegantly, but I think that would require a rewrite.